### PR TITLE
Wire up faults in fPoSt

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: chain/actors/actor_miner.go [hunk 1]
```diff
@@ -824,7 +824,7 @@ func (sma StorageMinerActor) DeclareFaults(act *types.Actor, vmctx types.VMConte
 	self.LastFaultSubmission = vmctx.BlockHeight()
 
 	nstate, aerr := vmctx.Storage().Put(self)
-	if err != nil {
+	if err != nil { // TODO: should be aerr
```
Comment (whyrusleeping) [inline chain/actors/actor_miner.go]: wanna also add FORK to this todo?

File: lib/sectorbuilder/scrub.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+func (sb *SectorBuilder) checkSector(sectorID uint64) error {
+	cache, err := sb.sectorCacheDir(sectorID)
+	if err != nil {
+		return xerrors.Errorf("getting sector cache dir: %w", err)
+	}
+
+	if err := assertFile(filepath.Join(cache, "p_aux"), 96, 96); err != nil {
+		return err
+	}
+	if err := assertFile(filepath.Join(cache, "sc-01-data-tree-r-last.dat"), (2*sb.ssize)-32, (2*sb.ssize)-32); err != nil {
```
Comment (whyrusleeping) [inline lib/sectorbuilder/scrub.go]: is this `01` there for all sectors?

Comment (magik6k) [reply lib/sectorbuilder/scrub.go]: If you have a 32G miner, would be great to verify

Comment (whyrusleeping) [reply lib/sectorbuilder/scrub.go]: I'll try that soon, just got new ram

Comment (magik6k) [reply lib/sectorbuilder/scrub.go]: Just confirmed

**Pull request merged**